swagger.json

```diff
@@ -6327,6 +6327,117 @@
+                        "type": "string",
+                        "enum": [
+                            "basic",
+                            "saml"
+                        ],
+                        "description": "The authentication type the user uses to authenticate. To use SAML your organization must have a configured SAML integration."
+                    }
+                }
+            }
+        },
+        "updateUserParam": {
+            "name": "user",
+            "description": "User properties. Note that when updating tagRoles, the complete list of tag roles must be supplied for the user. To delete all tag roles set the value to null or an empty array.",
+            "required": true,
+            "in": "body",
+            "schema": {
+                "type": "object",
+                "properties": {
+                    "name" : {
+                        "type": "string",
+                        "description": "The first and last name of the user.",
+                        "example": "Jane Doe"
+                    },
+                    "email": {
+                        "type": "string",
+                        "description": "The email address of this user.",
+                        "example": "user@company.com"
+                    },
+                    "organizationRole" : {
+                        "type": "string",
+                        "description": "The name of the role the user is assigned to for the organization. Leave this blank to give only tag access. This must match one of the preset roles or the name of a custom role in the organization. This value is case insensitive.",
+                        "example": "Read-Only Admin"
+                    },
+                    "tagRoles" : {
+                        "type": "array",
+                        "description": "If specified, the user will be a tag administrator for these tags. If left blank, the user has access to the entire organization.",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "tagName": {
+                                    "type": "string",
+                                    "description": "The name of the tag to grant this user access to.",
+                                    "example": "North Carolina Distribution"
+                                },
+                                "role" : {
+                                    "type": "string",
+                                    "description": "The name of the role the user is assigned to for this tag. This must match one of the preset roles or the name of a custom role in the organization. This value is case insensitive.",
+                                    "example": "Read-Only Admin"
+                                }
+                            }
+                        }
+                    },
+                    "authType": {
```

Comment:
Should we stick `authType` next to `name` and `email` to keep all the required params together?

Reply:
Makes sense to me.